packages/ember/tests/error_handler_test.js

```diff
@@ -1,9 +1,11 @@
 import Ember from 'ember';
 import { run } from 'ember-metal';
 import { DEBUG } from 'ember-env-flags';
+import TestAdapter from '@ember/test/adapter';
```

Comment:
This doesn't seem quite right, as CI is failing throughout all jobs.

Reply:
@locks This one I wasn't completely sure because of this `Ember.Test.adapter = ADAPTER`. Maybe that's the reason why CI complains. Should I revert it?

Reply:
Really updating live source code is not really the scope of #15723 and I'd be fine just updating inline docs here (and possibly assert/error messages).